frontends/mit-open/src/page-components/SearchDisplay/ProfessionalToggle.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from "react"
+import {
+  Collapse,
+  ToggleButton,
+  ToggleButtonGroup,
+  styled,
+} from "ol-components"
+import { RiBookOpenLine, RiBriefcase3Line } from "@remixicon/react"
+
+const StyledToggleButtonGroup = styled(ToggleButtonGroup)`
+  width: 100%;
+`
+const StyledButtonGroupContainer = styled.div`
+  margin-top: 10px;
+  margin-bottom: 10px;
+`
+const ExplanationContainer = styled.div`
+  margin: 10px;
+  font-size: 0.875em;
+`
+
+const StyledRiBookOpenLine = styled(RiBookOpenLine)`
+  height: 20px;
+  margin: 3px;
+`
+
+const StyledRiBriefcase3Line = styled(RiBriefcase3Line)`
+  height: 20px;
+  margin: 3px;
+`
+
+export const StyledToggleButton = styled(ToggleButton)(({ theme }) => ({
+  width: "100%",
+  height: "50px",
+  backgroundColor: theme.custom.colors.lightGray2,
+  borderLeft: "2px solid",
+  borderRight: "2px solid",
+  color: theme.custom.colors.darkGray2,
+  borderColor: theme.custom.colors.silverGrayLight,
+  "&.Mui-selected": {
+    backgroundColor: theme.custom.colors.white,
+    "&:hover": {
+      backgroundColor: theme.custom.colors.white,
+    },
+  },
+  "&:hover": {
+    backgroundColor: theme.custom.colors.white,
+  },
+}))
+
+const ProfessionalToggle: React.FC<{
+  professionalSetting: boolean | null | undefined | string
+  setParamValue: (_name: string, _rawValue: string | string[]) => void
+}> = ({ professionalSetting, setParamValue }) => {
+  const handleChange = (
+    event: React.MouseEvent<HTMLElement>,
+    newValue: string,
+  ) => {
+    setParamValue("professional", newValue)
+  }
+
+  if (professionalSetting !== true && professionalSetting !== false) {
+    professionalSetting = ""
+  } else {
+    professionalSetting = professionalSetting.toString()
+  }
+
+  return (
+    <StyledButtonGroupContainer>
+      <StyledToggleButtonGroup
+        value={professionalSetting}
+        exclusive
+        onChange={handleChange}
+        aria-label="text alignment"
+      >
+        <StyledToggleButton value="">View All</StyledToggleButton>
+        <StyledToggleButton value="false">
+          <StyledRiBookOpenLine /> Academic
+        </StyledToggleButton>
+        <StyledToggleButton value="true">
+          <StyledRiBriefcase3Line /> Professional
+        </StyledToggleButton>
+      </StyledToggleButtonGroup>
+      <Collapse in={professionalSetting === "false" ? true : false}>
+        <ExplanationContainer>
+          <b>Academic</b> - These courses cover a broad range of topics
```

Comment:
The text inside this 'container' should be "Content developed from MIT’s academic curriculum"

Reply:
@steven-hatch